packages/framework/src/Model/Administrator/Administrator.php

```diff
@@ -104,6 +104,13 @@ class Administrator implements UserInterface, Serializable, UniqueLoginInterface
      */
     protected $multidomainLoginTokenExpiration;
 
+    /**
+     * @var \DateTime|null
+     *
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    protected $rolesChangeAt;
```

Comment:
Firstly I used name `changeAt`, but then I realized, that I need to update time after refreshing adminstration roles, because I don't want to update administrator roles within every request. 
I call it in `AdministratorRolesChangedListener::onKernelRequest()` after roles refresh

Reply:
So I think `rolesChangeAt` is more precise, because I call in `Administrator::addRoles` and In `AdministratorRolesChangedListener`.
I haven't changed it